CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
  
 #### Fixes
 
+* [#1440](https://github.com/ruby-grape/grape/pull/1441): Remove pre-2.2.2 rubies from .travis.yml - [@jlfaber](https://github.com/jlfaber).
```

Comment:
That's a good point.  Although what we're actually doing is dropping support for 2.2.0 and 2.2.1 as well (i.e. Ruby < 2.2**.2**).  I'll adjust the PR.